akka-typed/src/main/scala/akka/typed/internal/ActorCell.scala

```diff
@@ -326,6 +315,23 @@ private[typed] class ActorCell[T](
         else unsafe.getAndAddInt(this, statusOffset, -1)
       }
     }
+
+    try {
+      unscheduleReceiveTimeout()
+      if (!isTerminated(status))
+        process()
+      scheduleReceiveTimeout()
+      afterProcess()
+    } catch {
+      case NonFatal(ex) ⇒
+        fail(ex)
+        afterProcess()
```

Comment:
This will not fly: `afterProcess` is not idempotent and here we don’t know which effects have already been performed. If for example the executor rejects execution then we’ll leave the _status field in a corrupt state.